qemu/Dockerfile

```diff
@@ -129,7 +128,7 @@ RUN echo http://mirror.yandex.ru/mirrors/alpine/v3.7/community >> /etc/apk/repos
         --enable-kvm \
         --enable-libiscsi \
         --enable-libnfs \
-        --enable-libssh2 \
+        --enable-libssh \
```

Comment:
Proof: https://wiki.qemu.org/ChangeLog/4.1#Build_Dependencies

Reply:
Let's add the link before the line.